packages/common/core-interfaces/src/events.ts

```diff
@@ -5,7 +5,9 @@
 
 /**
  * Base interface for event emitters.
- * @alpha
+ * @privateRemarks
+ * used by fluidframework/telemetry-utils public EventEmitterWithErrorHandling
+ * @public
```

Comment:
IIRC there are some typing annoyances with these event APIs which were derives from Node's event APIs. There are more TypeScript optimized variants in tree2 which have been working well: we may want to port to those, or at least reevaluate these before stabilizing them.

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/6595